### PR TITLE
Make Basset work on Vapor

### PR DESCRIPTION
**Pull request by tabacitu**
Thanks to @malek77z we might be able to add Vapor support really fast, fixing https://github.com/Laravel-Backpack/basset/issues/36

@malek77z I've asked @promatik to take a look at this and try it out. Thank you so much for submitting this 🙏

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "name": "backpack/basset",
     "description": "Dead-simple way to load CSS or JS assets only once per page, when using Laravel 10+.",
+    "version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (tabacitu) [reply composer.json:4]: We keep version information in tags instead of `composer.json`

File: src/config/backpack/basset.php [hunk 2]
```diff
@@ -5,11 +5,11 @@
     'dev_mode' => env('BASSET_DEV_MODE', env('APP_ENV') === 'local'),
 
     // disk and path where to store bassets
-    'disk' => 'public',
+    'disk' => env('BASSET_DISK')??'public',
```
Comment (promatik) [inline src/config/backpack/basset.php:8]:
```suggestion
    'disk' => env('BASSET_DISK', 'public'),
```

File: src/config/backpack/basset.php [hunk 3]
```diff
@@ -5,11 +5,11 @@
...
     'path' => 'basset',
 
     // use cache map file (.basset)
-    'cache_map' => true,
+    'cache_map' => env('BASSET_CACHE_MAP')??true,
```
Comment (promatik) [inline src/config/backpack/basset.php:12]:
```suggestion
    'cache_map' => env('BASSET_CACHE_MAP', true),
```

**Comment by promatik** [comment]
Moving to https://github.com/Laravel-Backpack/basset/pull/58 👌

**Pull request closed without merging**